js/app.js

```diff
@@ -1,35 +1,17 @@
 /* Set the width of the side navigation to 250px and the left margin of the page content to 250px and add a black background color to body */
 function openNav() {
   $u('#mySidenav')
-    .removeClass('closedsidenav')
     .addClass('opensidenav');
```

Comment:
the default behaviour of our sidenav is to be hidden, that is it is at -200px from the left. When we add the `opensidenav` class we translate that sidenav 200px to the right, so it show up, but if we remove that class it is going to be hidden again. We only have to *toggle* that class :-)

P.S: remember to keep it simple!